### DISCUSSION TITLE
search for all items that DONT have a tag?

### DISCUSSION THREAD

**Discussion by pisaniej** [netbox-community/netbox#8630] [Q&A]
Is there a way to easily search to find all items that do _not_ have a particular tag applied to them?

For example, given the inventory below:

| server | tags                      |
|--------|---------------------------|
| node1  | pizza, toast, chocolate     |
| node2  | broccoli, ice_cream, coffee |
| node3  | cake, broccoli, cookie      |

Let's say I hate `broccoli`, and wanted to find all items that are _not_ tagged with it, how would I do so?

desired result set:
```
node1
node3
```

Thank you.

**Comment by jeremystretch** [comment]
I'm wholly on board with your examples here. :laughing: 

To answer your question, you can append the [negation lookup](https://netbox.readthedocs.io/en/stable/rest-api/filtering/#lookup-expressions) to the tag filter to exclude specific tags. For example: `?tag__n=broccoli` will exclude all objects with the `broccoli` tag applied.

**Comment by maxzapp** [comment]
Hi, i am looking for such an excluding filter in the gui  (hopefully for all filter objects) is it already implemented i couldnt find it? (i am on nb 4.0.x)

**Reply by candlerb** [reply]
It's not in the GUI, but you can manually edit the URL in the browser, e.g. filter on `tag=foo` and then change it to `tag__n=foo`. Unfortunately, you can't Save such a filter (tested v4.1.6).

If this is something you need to do frequently with some specific tags then maybe you can set up [Custom Links](https://netboxlabs.com/docs/netbox/en/stable/customization/custom-links/) for this.

**Reply by maxzapp** [reply]
Ok, thats working for on tag as workaround but i think this should be a new feature in the gui to mix and match filters
